archive/restore.go

```diff
@@ -0,0 +1,262 @@
+package archive
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"math/big"
+	"os"
+
+	"github.com/0xPolygon/polygon-sdk/helper/common"
+	"github.com/0xPolygon/polygon-sdk/types"
+)
+
+var (
+	// Size of blocks to pass to WriteBlocks
+	chunkSize = 50
```

Comment:
Is there a possibility to customize this value to something higher?

50 blocks seems small, especially considering the time it takes to restore chains that have 100 000+ blocks

Reply:
It takes ~4mins to sync 100 000 blocks from 0 if we're using this `50` block batch value